README.md

```diff
@@ -69,3 +69,537 @@ A nearly-vanilla sufia7 app with the potential to be built out as GW ScholarSpac
 ```
    bin/rake db:migrate RAILS_ENV=development
 ```
+
+Installation with Apache, Tomcat 7, and Passenger <a id="prod-install"></a>
+------------
+Note: Solr, Fedora, PostgreSQL and the GW ScholarSpace application can all be deployed on different servers if desired.  If doing so, ensure that firewall ports are opened between the necessary servers and the GW ScholarSpace application server.  The instructions below will layout the environment as follows:
+
+Repository server: PostgreSQL, Tomcat, Solr, Java
+
+Application server: PostgreSQL, Apache2, Ruby, Rails, Passenger, Redis, Shibboleth
+
+### Repository server setup:
+
+### Dependencies
+
+* Install Java 8 for 14.04:
+
+        % sudo apt-add-repository ppa:webupd8team/java
+        % sudo apt-get update
+        % sudo apt-get install oracle-java8-installer
+        % sudo apt-get install oracle-java8-set-default
+
+* Install ubuntu 14.04 LTS package dependencies:
+
+        % sudo apt-get update
+        % sudo apt-get install git postgresql libpq-dev unzip clamav-daemon curl tomcat7 libcurl4-openssl-dev apache2-threaded-dev libapr1-dev libaprutil1-dev apache2-mpm-worker apache2-threaded-dev
+        
+* Create the directory structure
+
+        % mkdir /opt/install
+        % mkdir /opt/fedora
+        
+* Set up tomcat7
+
+  Configure Java setttings
+
+        % cd /opt/install
+        Retrieve `tomcat7` from `tomcat_conf/tomcat7` in the GitHub repo
```

Comment:
I'm confused by what's happening here.  Why are we switching to /opt/install?  Should the repo be cloned here?